config.json

```diff
@@ -1051,7 +1056,13 @@
         "name": "Twelve Days",
         "uuid": "64de1776-d5c6-43fe-9abf-7e3aa08ae342",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "arrays",
+          "array-transformations",
```

Comment:
Ok thanks. I did not know if it would be best to list all potential prerequisites.

In regards to transformations, I went based on the proof to determine this being a prerequisite.

Reply:
"Prerequisites" means that a student cannot work on this exercise until they unlocked those concepts. Because of that we do not want to list all potential prerequisites but instead answer the question "what is a minimal set of concepts someone needs to know to write a reasonable solution for this exercise".